G_channels.md

```diff
@@ -73,7 +73,12 @@ defmodule HelloPhoenix.UserSocket do
   use Phoenix.Socket
 
   channel "rooms:*", HelloPhoenix.RoomChannel
-  ...
+
+  transport :websocket, Phoenix.Transports.WebSocket
+
+  def connect(_, socket), do: {:ok, socket}
```

Comment:
How would you feel about changing `def connect(_, socket)` to `def connect(_params, socket)` to match the generated `UserSocket` module? I think it lets readers know what the argument contains, and still has the benefit of avoiding compiler warnings.